dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ResourcesServiceImpl.java

```diff
@@ -1871,6 +1874,61 @@ public Resource queryResourcesFileInfo(String userName, String fileName) {
         return (Resource) resourceResponse.getData();
     }
 
+    @Override
+    public Map<String, Object> deleteDataTransferData(User loginUser, Integer days) {
```

Comment:
We should avoid to use `Map<String, Object>` as result object, because it's not clearly.
```suggestion
    public DeleteDataTransferResponse deleteDataTransferData(User loginUser, Integer days) {
```

Reply:
I have changed it to `Result<Object>`, the same as other methods.

Reply:
> I have changed it to `Result<Object>`, the same as other methods.

The `Result<Object>` is also not clearly too, because it still do like 
```
Map<String, Object> data = new HashMap<>();
data.put("successList", successList);
data.put("failList", failList);
putMsg(result, Status.SUCCESS);
result.setData(data);
```
please see https://github.com/apache/dolphinscheduler/blob/9e0c9af1a5070b6a55750737aff2026233b26952/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessTaskRelationServiceImpl.java#L229

Reply:
I have added a new class `DeleteDataTransferResponse` to handle the method